modin/core/dataframe/algebra/groupby.py

```diff
@@ -51,14 +71,17 @@ def register(cls, map_func, reduce_func=None, **call_kwds):
             Function that takes query compiler and executes GroupBy aggregation
             with TreeReduce algorithm.
         """
-        if isinstance(map_func, str):
-
-            def build_fn(name):
-                return lambda df, *args, **kwargs: getattr(df, name)(*args, **kwargs)
-
-            map_func, reduce_func = map(build_fn, groupby_reduce_functions[map_func])
         if reduce_func is None:
             reduce_func = map_func
+
+        def build_fn(name):
```

Comment:
Why do we need such a wrapper?

Reply:
We need to build a callable function from an aggregation name. This is exactly what this function does.